### ISSUE TITLE
Few inconsistent parameters with paper.

### ISSUE THREAD

**Issue by adaxidedakaonang** [zhangchbin/RCIL#13]
Hi, recently I'm reproducing the ADE20K, and I noticed that in paper you mentioned the initial learning rate is 2e-2 and with cross_val, but in script your initial lr is 1e-2 and doesn't use cross_val. I'm little confused because it's hard to improve the performance for ADE20k.

**Comment by zhangchbin** [comment]
Hi, Thanks for your interests. Considering that there are few training iterations on VOC and our learnable branch has a weight of 0.5, so we try to set the initial learning rate as 0.02 rather than 0.01. However, for ADE20K, the training schedule is longer, so we just use the initial learning rate as 0.01. I guess the results of 2e-2 on ADE20K shouldn't be different.
For cross_val, we directly refer to training scripts of PLOP. Thank you very much. 
I will rerun the experiments on ADE20K and upload the training logs in few days.

**Comment by adaxidedakaonang** [comment]
Thank you very much. Besides, my initial step's performance of ADE20k_100-50 is 41.6%, is this normal or lower?  In the paper your final performance of first 100 classes is 42.3%. I'm not sure whether 41.6% is acceptable,  I guess the initial step's performance should be larger than 42.3% because the incremental step will cause the performance degradation. Did you remember the mIoU of initial step in ADE20k_100-50?

**Comment by zhangchbin** [comment]
hi, @adaxidedakaonang 
We found that high version pytorch can not reach the supposed performance due to the bug caused by InplaceABNSync, which has been fixed in our latest commitment. You can refer to our latest news. Sorry for the inconvenience caused.

**Comment by adaxidedakaonang** [comment]
Thanks for the update, the previous results were based on the single GPU device, but for now I will run it on multiple GPU devices, thank you very much!
